osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/SliderPlacementBlueprint.cs

```diff
@@ -89,6 +89,11 @@ public override void UpdateTimeAndPosition(SnapResult result)
 
                     HitObject.DifficultyControlPoint = nearestDifficultyPoint ?? new DifficultyControlPoint();
                     HitObject.Position = ToLocalSpace(result.ScreenSpacePosition);
+
+                    // Replacing the DifficultyControlPoint above doesn't trigger any kind of invalidation.
+                    // Without re-applying defaults, velocity won't be updated.
+                    // If this causes further issues, it may be better to copy the velocity p
```

Comment:
```suggestion
                    // If this causes further issues, it may be better to copy the velocity property instead.
```
I'm guessing?

Reply:
I meant to remove this, because I don't think it would help.